src/main/kotlin/io/github/mojira/arisa/modules/FutureVersionModule.kt

```diff
@@ -25,7 +25,9 @@ class FutureVersionModule(
 
             latestVersion!!.add()
             removeFutureVersions.forEach(::run)
-            resolveAsAwaitingResponse()
+            if (affectedVersions.size <= removeFutureVersions.size) {
+                resolveAsAwaitingResponse()
+            }
             addComment(CommentOptions(message))
```

Comment:
The message also should change. The message that is used always with this pr suggests that the ticket has been ARed
```
{panel:borderColor=orange}(!) Please do not mark _Unreleased Versions_ as affected. You don't have access to them yet.{panel}

_We have removed it and added the latest released version_

Please edit the *Affects Version/s* field and select the specific *Released Version* you were using when you encountered the reported issue.

If you can't find it in the list, please make sure that:
- You are in the *correct project* on the bug tracker.
- You were playing the *latest release version* or the *latest development version* of the game. 

~This issue has been temporarily closed as {color:#d04437}*Awaiting Response*{color}. Once the requested information has been delivered, the report will be reopened automatically.~

*Quick Links*:
📓 [Issue Guidelines|https://bugs.mojang.com/projects/MC/summary] -- 💬 [Community Support|https://discord.gg/58Sxm23] -- 📧 [Customer Support|https://help.minecraft.net/hc/en-us/requests/new] -- ✍️ [Feedback and Suggestions|https://feedback.minecraft.net/] -- 📖 [Game Wiki|https://minecraft.gamepedia.com/Minecraft_Wiki]
```

We do have the panel on its own and I suggest just usning the message called Panel - Future Version when not resolving as AR:
```
{panel:borderColor=orange}(!) Please do not mark _Unreleased Versions_ as affected. You don't have access to them yet.{panel}
```